traffic_portal/app/src/common/modules/table/cacheGroupParameters/TableCacheGroupParametersController.js

```diff
@@ -32,10 +44,6 @@ var TableCacheGroupParametersController = function(cacheGroup, cacheGroupParamet
 			);
 	};
 
-	$scope.refresh = function() {
```

Comment:
its because i moved a bunch of functions into the "parent" controller and those functions are still available to this controller.

```
// extends the TableParametersController to inherit common methods
	angular.extend(this, $controller('TableParametersController', { parameters: parameters, $scope: $scope }));
```

Reply:
a lot of this PR is moving duplicate functions into parent controllers

Reply:
oh, okay